iio-private.h

```diff
@@ -34,7 +34,7 @@
 #define is_little_endian() (__BYTE_ORDER__ == __ORDER_LITTLE_ENDIAN__)
 #endif
 
-#define BIT(x) (1 << (x))
+#define BIT(x) (1ull << (x))
```

Comment:
Seems odd to make this 64-bit but not the other `BIT` macros.

Reply:
I only make the `BIT` macro able to create 64-bit masks, the `BIT_MASK` and `BIT_WORD` still need to work on 32-bit words. I *could* make them work with 64-bit words, but that means changing the `iio_channels_mask` struct, which is a bit outside this PR.

Reply:
Thanks for reviewing btw.